### ISSUE TITLE
post_opf not defined

### ISSUE THREAD

**Issue by csaiedu** [lanl-ansi/PowerModels.jl#775]
When trying to examples on the tutorial I am facing this issue with Version 0.18.0

pm = build_generic_model("case3.m", ACPPowerModel, PowerModels.post_opf)

ERROR: UndefVarError: post_opf not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base .\Base.jl:26
 [2] top-level scope
   @ REPL[27]:1

  c52e3926] Atom v0.12.31
  [336ed68f] CSV v0.8.4
  [a93c6f00] DataFrames v1.0.1
  [60bf3e95] GLPK v0.14.8
  [2030c09a] InfrastructureModels v0.6.0
  [b6b21f68] Ipopt v0.6.5
  [4076af6c] JuMP v0.21.7
  [e5e0dc1b] Juno v0.8.4
  [c36e90e8] PowerModels v0.18.0
  [f4570300] SDDP v0.3.14

Any help please for a beginner with PowerModels. Thanks.

**Comment by ccoffrin** [comment]
Hi @csaiedu, some time ago the `post_*` functions were renamed to `build_*` so `build_opf` should work in your example.  I am wondering what tutorial you are referencing. I checked this documentaiton,

https://lanl-ansi.github.io/PowerModels.jl/stable/quickguide/#Building-PowerModels-from-Network-Data-Dictionaries

And it appears to be up to date.

**Comment by csaiedu** [comment]
Thanks for the answer. I was looking annoyingly at the wrong manual: Getting Started · PowerModels (lanl-ansi.github.io) I see that google searches points to prior versions indeed, here it’s v0.7 I have updated my code and all works fine now. Kind regards From: Carleton CoffrinSent: 24 April 2021 14:53To: lanl-ansi/PowerModels.jlCc: csaiedu; MentionSubject: Re: [lanl-ansi/PowerModels.jl] post_opf not defined (#775) Hi @csaiedu, some time ago the post_* functions were renamed to build_* so build_opf should work in your example. I am wondering what tutorial you are referencing. I checked this documentaiton,https://lanl-ansi.github.io/PowerModels.jl/stable/quickguide/#Building-PowerModels-from-Network-Data-DictionariesAnd it appears to be up to date.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
